src/components/navigation/header/config/showUnderMoreConstants.ts

```diff
@@ -0,0 +1,14 @@
+import { breakpoints } from 'src/themes';
+
+export type ShowUnderMoreBreakpoint = `(max-width: ${number}px)`;
+
+const minWidthMore = 500;
```

Comment:
This introduces an artificial breakpoint that's not part of our design system

Reply:
Yes correct. I can change that if you see it as a blocker but to me it looks much nicer since there is no wasted space on the larger mobiles. Wdyt to make an exception here?

Reply:
I would not add exceptions to breakpoints. (This does not mean to say it looks better). I would use what we have and have UX push extending the breakpoint concept if they see the need for it but for now go with the ones we have)

Reply:
done. Looks a bit sad from 500-767px :( 

<img width="887" alt="Bildschirmfoto 2024-08-07 um 16 00 57" src="https://github.com/user-attachments/assets/bb44eb71-25d6-461d-b81a-570663bfa2fb">